### ISSUE TITLE
apoc.uuid.install not setting uuid on new nodes

### ISSUE THREAD

**Issue by JVemmer** [neo4j-contrib/neo4j-apoc-procedures#2137]
## Guidelines

Please note that GitHub issues are only meant for bug reports/feature requests. If you have questions on how to use APOC, please ask on [the Neo4j Discussion Forum](https://community.neo4j.com/c/neo4j-graph-platform/procedures-apoc) instead of creating an issue here.

## Expected Behavior (Mandatory)
That `apoc.uuid.install` eventually sets a uuid on a newly created node.

## Actual Behavior (Mandatory)
It currently does not set it on newly created nodes.

## How to Reproduce the Problem

1. Install Neo4j 4.1.10 on Ubuntu 20.04, and install APOC apoc-4.1.0.9-all.jar in the plugs folder.
2. Set `apoc.uuid.enabled=true` in either apoc.conf or neo4j.conf (tried both) as per documentation.
3. Create a unique constraint in the database using `CREATE CONSTRAINT ON (person:Person) ASSERT person.uuid IS UNIQUE;`
4. Create the uuid handler using either `CALL apoc.uuid.install("Person", {addToExistingNodes: false, uuidProperty: 'uuid', addToSetLabels: true}) YIELD label, installed, properties, batchComputationResult
RETURN label, installed, properties, batchComputationResult` or the same, with "addToExistingNodes" set to true (both displayed same issue).
5. Create/Merge a new node with the Person label via either `CREATE (n:Person {name: "John Doe") RETURN n` or `MERGE (n:Person {name: "John Doe"}) RETURN n` - The uuid is not set here as expected.
6. Find the person you just created, using `MATCH (n:Person {name: "John Doe"}) RETURN n` - No uuid present, even hours later (to account for slowness).

### Simple Dataset (where it's possible)

Terminating separate cypher queries via semi-colon. 
```
CREATE CONSTRAINT ON (person:Person) ASSERT person.uuid IS UNIQUE;

CALL apoc.uuid.install("Person", {addToExistingNodes: false, uuidProperty: 'uuid', addToSetLabels: true}) YIELD label, installed, properties, batchComputationResult
RETURN label, installed, properties, batchComputationResult; 

CREATE (n:Person {name: "John Doe") RETURN n;

MATCH (n:Person {name: "John Doe"}) RETURN n;
```

### Steps (Mandatory)

Do either: Follow the reproduction steps above or follow the steps in the documentation [https://neo4j.com/labs/apoc/4.1/overview/apoc.uuid/apoc.uuid.install/](https://neo4j.com/labs/apoc/4.1/overview/apoc.uuid/apoc.uuid.install/). Both exhibit the issue.

## Specifications (Mandatory)

Currently used versions

### Versions

  - OS: Ubuntu 20.04
  - Neo4j: 4.1.10
  - Neo4j-Apoc: 4.1.0.9-all

**Comment by JVemmer** [comment]
An update to this: I upgraded the environment where I tested this to Neo4j 4.3 instead of 4.1. 4.3 (and the related apoc 4.3.0.0) did **not** exhibit the issue. So it appears fixed in 4.3.

**Comment by vga91** [comment]
It should have been fixed for [4.4](https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/2350) and [5.x](https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/2351) (which are currently the two supported versions).

In case there are still problems, feel free to reopen the issue
